content/articles/Developing-simple-Blockchain-app-In-flutter/index.md

```diff
@@ -0,0 +1,216 @@
+
+In this article, we will write about creating a blockchain flutter application.
+### Introduction
+ A flutter application is a  mobile app dart code that runs on iOS and Android.  Flutter uses Google's Skia library to draw widgets in UIs, which gives it high performance. Flutter also has its advanced framework that lets developers create mobile apps easily and quickly
```

Comment:
The first part of that sentence needs some work. It has some grammar issues. 
Also, it's awesome that you mentioned Skia 💯